deployment-manager/src/main/java/com/example/deploymentmanager/DeploymentManagerDemo.java

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2019 Google LLC. This software is provided as-is, without warranty or representation for any use or purpose. Your use of it is subject to your agreement with Google.
```

Comment:
Usually on a seperate line.  Shouldn't this be apache?

Reply:
Done.

Reply:
Yes, Done.